lib/reporters/cli/index.js

```diff
@@ -88,7 +88,8 @@ PostmanCLIReporter = function (emitter, reporterOptions, options) {
             run.executions &&
             run.executions.length &&
             run.executions[0].response &&
-            run.executions[0].response.text()) {
+            run.executions[0].response.text() &&
+            run.transfers.responseTotal < 10485760) {
```

Comment:
Points while using the numbers like these
- never hardcode them in the logic but always use config.
- never use the whole final number but how you reached at that number example `1024 * 1024 * 10`. This makes reading the code much easier.
- always add a comment on what that number signifies with correct unit, example // 10 MB